webapp/javascript/components/TimelineChartWrapper.tsx

```diff
@@ -37,6 +37,8 @@ type TimelineChartWrapperProps = {
     left?: Marking;
     right?: Marking;
   };
+
+  selectionType: 'single' | 'double';
```

Comment:
Can you comment what this field does? Preferrably using tsdoc https://tsdoc.org/pages/tags/example/

Reply:
added comment on it

Reply:
>selection looks different

How different? Consider that this is the public api of that component. Ideally one does not have to look into the implementation to figure out what it does.

Reply:
updated comment